packages/mobile/src/send/SendAmount.tsx

```diff
@@ -379,9 +381,7 @@ const styles = StyleSheet.create({
     marginLeft: 8,
   },
   amount: {
-    ...fontStyles.regular,
```

Comment:
We can't keep our custom font?

Reply:
oo my bad! i think the custom font and the regular font looked so similar i didnt realize this line stayed! will re-add

Reply:
oh i see, actually. the custom had a set line height that was messing up the display of the amount. also, since the line height changes as the font gets smaller, i think we should take it out. it looks like it still retains font family

Reply:
yes let's keep the font and set `lineHeight: undefined` here then.

Reply:
sounds good!